lib/Config.js

```diff
@@ -263,6 +263,13 @@ class Config {
             dataPath,
             metadataPath,
         };
+
+        if (config.kinetic && Array.isArray(config.kinetic)
+            && process.env.S3DATA === 'kinetic') {
+            this.kinetic = {};
```

Comment:
You can write it like that :)
```
const tab = config.kinetic[0].split(':');
this.kinetic = {
    host: tab[0],
    port: tab[1],
};
```
You avoid one split, and you do not add property to the object.